book/src/math.tex

```diff
@@ -837,8 +837,530 @@ \subsection{Common Usage}
+$10~\frac{m}{s} \cdot 12~s = 120~m$
+\end{example}
+Here we also need to make sure that the units are written in roman font.
+The above example should be then written as
+\begin{example}[examplewidth=6cm]
+$10~\frac{\mathrm{m}}{\mathrm{s}} 
+  \cdot 12~\mathrm{s} 
+  = 120~\mathrm{m}$
+\end{example}
+Hopefully these examples illustrate why a better method is needed.
+
+\subsection{Basic commands of the \pai{siunitx} package}
+
+The basic commands of \pai{siunitx} package are
+\begin{lscommand}
+  \ci{num}[options][number]\\
+  \ci{unit}[options][unit]\\
+  \ci*{qty}[options:o ! number:m ! unit:m] 
+\end{lscommand}
+The \ci{num} command typesets a number, \ci{unit} typesets a unit, and \ci{qty}
+typesets a quantity, i.e.\ a number followed by a unit. All of the commands
+provide optional \carg{options} argument, which is a comma delimited key value
+list that can influence the output of the command. The \carg{unit} argument can
+be provided in one of two styles, either \enquote{literal} or
+\enquote{interpreted}.
+
+In literal mode units are entered as string of letters 
+\begin{example}
+\unit{kg} is a unit,
+\qty{10}{m/s} is a quantity
+\end{example}
+Spaces in literal mode are ignored. If you want to insert a product of units,
+use either dot \ai{.} or tilde \ai{\~}.
+\begin{example}
+\unit{N} is 
+not \unit{kg m / s^2},
+but \unit{kg.m / s^2}
+or \unit{kg~m / s^2}
+\end{example}
+
+On the other hand, in interpreted mode units are entered using predefined
+macros
+\begin{example}
+\unit{\kilo\gram} is a unit,
+\qty{10}{\metre\per\second}
+is a quantity
+\end{example}
+At first glance this may seem less convenient, but this enables the package to
+recognise the logical structure of the units, which means that the formatting
+can be changed on the fly.
+\begin{example}[examplewidth=0.32\linewidth]
+Which one do you prefer? \\
+$\unit{\newton} = \unit{
+  \kilo\gram\metre\per\square\second}$ \\
+$\unit{\newton} = \unit[per-mode = fraction]{
+  \kilo\gram\metre\per\second\squared}$ \\
+$\unit{\N} = \unit[per-mode = symbol]{
+  \kg\m\per\s\tothe{2}}$
+\end{example}
+The commands \ci{gram}, \ci{kilo}, \ci{per} are rather self explanatory. As you
+can see we have also used the \cargv{per-mode} option to influence the unit
+style. While the command \ci{tothe} typesets arbitrary superscripts, the
+command \ci{of} typesets arbitrary subscripts, known as qualifiers.
+\begin{example}[examplewidth=0.32\linewidth]
+\unit{\kg\of{mol}}, \unit{\bel\of{i}}
+\end{example}
+
+Note that the \ci{per} command only works on the next unit. If we want it to
+apply to all following units we may use \cargv{sticky-per} option.
+\begin{example}[examplewidth=0.4\linewidth]
+The unit of thermal conductivity is
+\unit{\watt\per\metre\per\kelvin}
+or equivalently
+\unit[sticky-per]{\m\kg\per\s\cubed\K}. 
+\end{example}
+
+Sometimes you may want to make the units use a different colour so they stick
+out more. This can be done using \cargv{unit-color}. Options
```

Comment:
```suggestion
Sometimes you may want to make the units use a different colour so they figure more prominenty. This can be done using \cargv{unit-color}. Options
```